modules/models/execution/execution_model.hpp

```diff
@@ -22,33 +22,58 @@ using dynamic::DynamicModelPtr;
 using dynamic::Trajectory;
 using dynamic::State;
 
+
+enum ExecutionStatus : unsigned int {
+  VALID = 0,
+  INVALID = 1
+};
+
+
 class ExecutionModel : public commons::BaseType {
  public:
   explicit ExecutionModel(modules::commons::ParamsPtr params) :
     BaseType(params),
-    last_trajectory_() {}
+    last_state_(),
+    last_trajectory_(),
+    execution_status_(ExecutionStatus::VALID) {}
 
   ExecutionModel(const ExecutionModel& execution_model) :
     BaseType(execution_model.GetParams()),
-    last_trajectory_(execution_model.GetLastTrajectory()) {}
+    last_state_(execution_model.GetLastState()),
+    last_trajectory_(execution_model.GetLastTrajectory()),
+    execution_status_(execution_model.GetExecutionStatus()) {}
 
   virtual ~ExecutionModel() {}
 
+  State GetLastState() const { return last_state_; }
```

Comment:
maybe call this GetExecutedState() ? last state could be unclear

Reply:
sounds good!